config/htaccess.yaml

```diff
@@ -4894,38 +4873,17 @@ to: '/core/sharded-cluster-config-servers'
 type: 'redirect'
 code: 301
 outputs:
-  - 'before-v3.0'
----
-from: '/core/sharding-chunk-migration'
-to: '/core/sharding-balancer-administration'
-type: 'redirect'
-code: 301
-outputs:
-  - 'before-v3.0'
----
-from: '/core/sharding-chunk-splitting'
-to: '/core/sharding-balancer-administration'
-type: 'redirect'
-code: 301
-outputs:
-  - 'before-v3.0'
----
-from: '/core/sharding-introduction'
-to: '/sharding'
-type: 'redirect'
-code: 301
-outputs:
-  - 'before-v3.0'
+  - 'after-v3.0'
 ---
-from: '/core/sharding-shard-key-indexes'
-to: '/core/sharding-shard-key'
+from: '/core/sharding-balancer-administration'
+to: '/core/sharding-chunk-migration'
 type: 'redirect'
 code: 301
 outputs:
   - 'before-v3.0'
 ---
-from: '/tutorial/choose-a-shard-key'
-to: '/core/sharding-shard-key'
+from: '/sharding'
+to: '/core/sharding-introduction'
```

Comment:
So, there is no such /core/sharding-introduction in v3.0

This version of the redirects are published, so could you test the redirect changes and make apppropriate changes and do another pull?